tests/sort-demo/TestBoard.pas

```diff
@@ -88,15 +88,41 @@ procedure TestTBoard.TestGenerateNegativeNumberOfData;
 end;
 
 procedure TestTBoard.TestSwapZeroAndOne;
+var
+  temp0: Integer;
+  temp1: Integer;
 begin
   // TODO: [TeamA] Zweryfkować swap indeksów 0 i 1
   // Uwaga! Najpierw trzeba wypełnić dane
+
+  FBoard.GenerateData(5);
+
+  temp0 := FBoard.Data[0];
+  temp1 := FBoard.Data[1];
+
+  FBoard.Swap(0, 1);
+
+  Check(temp0 = FBoard.Data[1], 'Wartości się nie zamieniły');
```

Comment:
Można by dać różne message, tak jak jeden warunek przejdzie a drugi nie to nie bedziemy wiedzieć, który nie przeszedł

Reply:
👍  Brawo! Jak wspominałem asercja powinna być 1 !!! Ale ja to tutaj zrobić w jednym Check-u?  

Jeśli chodzi o cytowanie kodu przez GitrHub-a to można też tak:
https://github.com/bogdanpolak/akademia/blob/660c3d9cbd56f88120e78673609e46edc318e213/tests/sort-demo/TestBoard.pas#L90-L107

Reply:
Moje uwagi:
* TODO usuwamy - bo nie chcemy umarłych komentarzy w kodzie
* Za duże przestrzenie pionowe w kodzie (kod powinien zająć 12 linijek, a zajmuje 18) przy Code Review to duża różnica
  * Pionowe odstępy trzeba robić, ale w bardziej skomplikowanych metodach i to głównie, aby pokazać bloki do dekompozycji (ekstrakcji metod)